docs/comparison/nbdb.md

```diff
@@ -103,53 +100,39 @@ print(reg.get_params())
 y_pred = reg.predict(X_test)
 mean_squared_error(y_test, y_pred)
 """
-Path('model.py').write_text(model)
+
+data_nb = jupytext.reads(data, fmt='py:percent')
+model_nb = jupytext.reads(model, fmt='py:percent')
+
+nbformat.write(data_nb, 'data.ipynb')
+nbformat.write(model_nb, 'model.ipynb')
 ```
 
-## Pipeline declaration
+## Executing notebooks
 
-Create a pipeline using [Ploomber](https://docs.ploomber.io/en/latest/). Each experiment will create an output `.ipynb` file.
+Using [ploomber-engine](https://ploomber-engine.readthedocs.io/en/latest/quick-start.html), each experiment will create an output `.ipynb` file.
 
 ```{code-cell} ipython3
```

Comment:
I think we can hide the cell output as default since it's very long and makes it hard to read. If someone wants to see the output he can hit the `Click to show`. [Here are some options](https://jupyterbook.org/en/stable/interactive/hiding.html) that might be relevant.

Reply:
`Using ploomber-engine` : You can specifically mention the method (`execute_notebook`) that is doing the conversion.
Or move this import `from ploomber_engine import execute_notebook` to this cell so user can understand which method is from ploomber-engine.
In general imports can be done in the relevant cells